crowdsourcing/crowdsourcing_tutorial.py

```diff
@@ -295,15 +273,16 @@ def encode_text(text):
 sklearn_model = LogisticRegression(solver="liblinear")
 sklearn_model.fit(train_vectors, probs_to_preds(Y_train_prob))
```

Comment:
We can use label_model.predict instead of label_model.predict_proba followed by probs_to_preds